File: docs-old/notintoc/codesnippet/CSharp/how-to--use-the-animation-region-of-the-status-bar_1.cs
        void AnimationExample()
        {
            IVsStatusbar statusBar = 
                (IVsStatusbar)GetService(typeof(SVsStatusbar));
            object icon = 
                (short)Microsoft.VisualStudio.Shell.Interop.Constants.SBAI_General;

            // Display the animated Visual Studio icon in the Animation region.
            statusBar.Animation(1, ref icon);

            System.Windows.Forms.MessageBox.Show(
                "Click OK to end status bar animation.");

            statusBar.Animation(0, ref icon);
        }

File: docs-old-1102/misc/codesnippet/VisualBasic/how-to-use-the-animation-region-of-the-status-bar_1.vb
    Private Sub AnimationExample()
        Dim statusBar As IVsStatusbar = CType(GetService(GetType(SVsStatusbar)), IVsStatusbar)
        Dim icon As Object = CShort(Fix(Microsoft.VisualStudio.Shell.Interop.Constants.SBAI_General))

        ' Display the animated Visual Studio icon in the Animation region.
        statusBar.Animation(1, icon)

        System.Windows.Forms.MessageBox.Show("Click OK to end status bar animation.")

        statusBar.Animation(0, icon)
    End Sub